base/libgit2/libgit2.jl

```diff
@@ -494,14 +494,14 @@ a `GitError`. This is roughly equivalent to the following command line statement
 function rebase!(repo::GitRepo, upstream::AbstractString="", newbase::AbstractString="")
     with(head(repo)) do head_ref
         head_ann = GitAnnotated(repo, head_ref)
-        upst_ann  = if isempty(upstream)
+        upst_ann = if isempty(upstream)
             brn_ref = LibGit2.upstream(head_ref)
-            if brn_ref === nothing
+            if isnull(brn_ref)
                 throw(GitError(Error.Rebase, Error.ERROR,
                                "There is no tracking information for the current branch."))
             end
             try
-                GitAnnotated(repo, brn_ref)
+                GitAnnotated(repo, get(brn_ref))
```

Comment:
My point wasn't `LibGit2.get` is bad, that much is obvious. It was the "does the LibGit2 module currently import `Base.get` ?" question, because the yes/no answer to that determines whether or not this code is correct right now.